src/demos/demo.react.jsx

```diff
@@ -24,7 +24,7 @@ class MyEventDemoReact extends React.Component {
   render() {
     return (
       <axa-button onClick={this.handleClick}>
-        {this.state.isToggleOn ? 'ON' : 'OFF'}
+        <span>{this.state.isToggleOn ? 'ON' : 'OFF'}</span>
```

Comment:
@zuzana 
The real issue is that we don't support incremental rendering of any web component right now.
Adding `<span>` isn't the proper way to deal with it. All necessary features are specified here #147. And for easy testing support of interactive demos #183

Reply:
Correct place to fix all of those is:
https://github.com/axa-ch/patterns-library/blob/master/src/components/_abstract/component-types.js

Reply:
Agree for when it goes production ready. As we are still beta, this quick fix will enhance the demo and therefore for me is ok to merge. The dist folder however should be reset and we will make a deploy as soon as this is merged in order to update the dist and docs folder.

Reply:
changes to dist folder reverted.